src/KeyUnit.cpp

```diff
@@ -296,320 +296,320 @@ QString KeyUnit::getKeyName( int keyCode, int modifierCode )
 
 void KeyUnit::setupKeyNames()
 {
-    mKeys[0x01000000]=QString("Escape");
```

Comment:
\<aside>A whole load of `tr(`...`)` to go in here for __I18n__ I guess... :smirk: \</aside>